src/docs/guides/assets-manager/README.md

```diff
@@ -26,6 +26,8 @@ What to do if you change an SCSS, JS, or TS/TSX file in any of Orchard Core's pr
 1. Make sure you completed the above "Prerequisites" steps.
 2. Run `yarn build -gr` from the command line in the root of the repository. This will build all changed assets.
 
+Alternatively, if you make a lot of changes during development that you want to test quickly, you don't need to run the full build every time. Instead, use `yarn watch` to automatically build assets when you save a file. For this, run `yarn watch -n module-name`, where `module-name` is the `name` property you can find in the `package.json` file of the given project's `Assets` folder. E.g., for the Audit Trail module it's `@orchardcore/audittrail`.
```

Comment:
No, this is not accurate. You need to take the name of the asset name in the Assets.json file.
`yarn watch -n audittrail`

Reply:
Fixed this documentation

Reply:
Thanks!